src/frontend/org/voltdb/importclient/kafka/KafkaStreamImporter.java

```
@@ -330,6 +334,7 @@ public void configure(Properties p) {
         //These are defaults picked up from kafka we save them so that they are passed around.
         m_fetchSize = Integer.parseInt(m_properties.getProperty("fetch.message.max.bytes", "65536"));
         m_consumerSocketTimeout = Integer.parseInt(m_properties.getProperty("socket.timeout.ms", "30000"));
+        m_backpressureSleepMs = Integer.parseInt(m_properties.getProperty("backpressure.sleep.ms", "50"));
```

I would rather get startup failure than ignore exception because of bad deployment.